### PR TITLE
PIM-4236: re-work attribute option csv import

### PR DESCRIPTION
**Pull request by solivier**

### CODE REVIEW

File: src/Pim/Bundle/InstallerBundle/Resources/config/fixtures_jobs.yml [hunk 1]
```diff
@@ -14,16 +14,6 @@ jobs:
             circularRefsChecked: true
             uploadAllowed: true
 
-    fixtures_category_yml:
```
Comment (skeleton) [inline src/Pim/Bundle/InstallerBundle/Resources/config/fixtures_jobs.yml]: why remove this ?

Comment (solivier) [inline src/Pim/Bundle/InstallerBundle/Resources/config/fixtures_jobs.yml]: because it's not used anymore

Comment (nidup) [inline src/Pim/Bundle/InstallerBundle/Resources/config/fixtures_jobs.yml]: yep we get ride of all unused format ;)

**Comment by nidup** [comment]
think about do a PR for EE and update the changelog (to say we removed these yml stuff there is already a line for that) then GTM

**Pull request merged**